src/util/string.h

```diff
@@ -52,4 +54,16 @@ NODISCARD inline bool ValidAsCString(const std::string& str) noexcept
     return str.size() == strlen(str.c_str());
 }
 
+/**
+ * Locale-independent version of ToString
+ */
+template <typename T>
+std::string ToString(const T t)
```

Comment:
Not really necessary for the simple types in all current call sites, but for future-proofing, `const T& t` is better to avoid copying expensive types.

Reply:
And should be inlined anyway, so the extra reference for simple types should get optimised out

Reply:
Yeah did not originally due to the definition of `std::to_string`, but I agree it makes sense for future-proofing.